src/createServer.js

```diff
@@ -1,3 +1,61 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = [];
+    const input = req.url.split('?');
+    const params = new URLSearchParams(input[1]);
```

Comment:
The code assumes that the URL will always contain a query string. If the URL does not contain a '?', `input[1]` will be undefined, leading to an error when creating `URLSearchParams`. Consider adding a check to ensure `input[1]` is defined before using it.